common/config/rush/pnpm-lock.yaml

```diff
@@ -3,15 +3,15 @@ dependencies:
   '@microsoft/rush-stack-compiler-3.4': 0.3.2
   '@microsoft/teams-js': 1.3.0-beta.4
   '@microsoft/tsdoc': 0.12.14
-  '@pnpm/link-bins': 1.0.3
+  '@pnpm/link-bins': 1.0.3_@pnpm+logger@1.0.2
   '@pnpm/logger': 1.0.2
   '@rush-temp/api-documenter': 'file:projects/api-documenter.tgz'
   '@rush-temp/api-documenter-test': 'file:projects/api-documenter-test.tgz'
   '@rush-temp/api-extractor': 'file:projects/api-extractor.tgz'
   '@rush-temp/api-extractor-lib1-test': 'file:projects/api-extractor-lib1-test.tgz'
   '@rush-temp/api-extractor-lib2-test': 'file:projects/api-extractor-lib2-test.tgz'
   '@rush-temp/api-extractor-lib3-test': 'file:projects/api-extractor-lib3-test.tgz'
-  '@rush-temp/api-extractor-model': 'file:projects/api-extractor-model.tgz'
+  '@rush-temp/api-extractor-model': 'file:projects/api-extractor-model.tgz_tslint@5.12.1'
```

Comment:
@zkochan - this is the only thing that's concerning. Does this seem right?

Reply:
yes. This is correct. It happens when the tarball has a peer dependnency on tslint